lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -38,6 +38,14 @@ def for(resource)
   end
 
   class IdResolver < HammerCLIForeman::IdResolver
+    require 'hammer_cli_katello/search_options_creators'
```

Comment:
This seem strange to need to do here in general and to do it at least at the top of the file if required.